infra/docker-evm-blockhain/readme.md

```diff
@@ -0,0 +1,148 @@
+To create a Docker container that automates the creation of a private Ethereum blockchain using the Proof of Authority consensus algorithm, follow these steps:
+
```

Comment:
This readme assumes there are no Dockerfile, genesis.json, start.sh, etc and give instructions to create those files. 

However, when someone comes to this repo, these files are already available in the repository. Hence, shouldn't we create the instructions on how to create the nodes by using the existing files?

Reply:
Update the README file to explain the steps on how to run the private blockchain using the provided Docker image. Remove the genesis.json file from the directory.